Story/StoryHooks.cs

```diff
@@ -97,9 +97,22 @@ private void StoryHooks()
             On.Menu.SlugcatSelectMenu.SetChecked += SlugcatSelectMenu_SetChecked;
             On.Menu.SlugcatSelectMenu.GetChecked += SlugcatSelectMenu_GetChecked;
             On.Menu.PauseMenu.SpawnExitContinueButtons += PauseMenu_SpawnExitContinueButtons;
+
+            On.VoidSpawnKeeper.AddOneSpawn += VoidSpawnKeeper_AddOneSpawn;
         }
 
 
+        private void VoidSpawnKeeper_AddOneSpawn(On.VoidSpawnKeeper.orig_AddOneSpawn orig, VoidSpawnKeeper self)
+        {
+            if (isStoryMode(out var _) && !OnlineManager.lobby.isOwner)
```

Comment:
feels a bit silly to only allow host to spawn them

also this desynced bodychunk issue _will_ affect other gamemodes too so it doesn't make sense to gate it behind story